Team6220_PowerPlay/src/main/java/org/firstinspires/ftc/team6220_PowerPlay/testclasses/OnePlusFiveTest.java

```diff
@@ -0,0 +1,11 @@
+package org.firstinspires.ftc.team6220_PowerPlay.testclasses;
+import com.qualcomm.robotcore.eventloop.opmode.Autonomous;
+import org.firstinspires.ftc.team6220_PowerPlay.BaseAutonomous;
+
+@Autonomous(name = "hopium", group = "Worlds Autos")
```

Comment:
Make sure this name matches so you can use it in world without being confused. Also make sure you disable any test classes you won't be using at world after you're done with all the testing.